opentelemetry-exporter-geneva/geneva-uploader/src/payload_encoder/otlp_encoder.rs

```diff
@@ -32,80 +31,109 @@ pub struct OtlpEncoder {
     schema_cache: SchemaCache,
 }
 
+impl Default for OtlpEncoder {
```

Comment:
Why is this being added?

Reply:
I got lint error during changes, for this to be added. don't see it any more. Removed.

Reply:
Oh there is error:
error: you should consider adding a `Default` implementation for `OtlpEncoder`
  --> opentelemetry-exporter-geneva/geneva-uploader/src/payload_encoder/otlp_encoder.rs:35:5
   |
35 | /     pub fn new() -> Self {
36 | |         OtlpEncoder {
37 | |             schema_cache: Arc::new(RwLock::new(HashMap::new())),
38 | |         }
39 | |     }
   | |_____^

Reply:
That's because `OtlpEncoder` struct is public. I think the better thing to do here is to mark it `pub(crate)`.

Reply:
oh yes Thanks.